docs/Build-A-Portable-Bottle.md

```diff
@@ -0,0 +1,71 @@
+# Building A Portabe Bottle for Linux
+
+## Introduction
+
+The bottles for `binutils`, `patchelf` and `zlib` are special. Because they are installed before `glibc`, they must be able to run on host systems using an older version of `glibc`. Linuxbrew supports host systems with `glibc` 2.13 and newer. We use Debian 7 (wheezy) with glibc 2.13 to build portable bottles.
```

Comment:
```suggestion
The bottles for `binutils`, `patchelf` and `zlib` are special. Because they are installed before `glibc`, they must be able to run on host systems using an older version of `glibc`. Homebrew on Linux supports host systems with `glibc` 2.13 and newer. We use Debian 7 (wheezy) with glibc 2.13 to build portable bottles.
```